static/js/resize.js

```diff
@@ -23,7 +23,7 @@ function size_blocks(blocks, usable_height) {
     _.each(blocks, function (block) {
         var ratio = block.real_height / sum_height;
         ratio = confine_to_range(0.05, ratio, 0.85);
```

Comment:
Do we also want to change `ratio` here?

More generally, can you write a detailed commit message explaining why this is correct?